pandas/tests/indexes/test_interval.py

```diff
@@ -491,6 +594,188 @@ def testcontains(self):
         assert not i.contains(20)
         assert not i.contains(-20)
 
+
+    @pytest.mark.xfail(reason="new indexing tests for issue 16316")
```

Comment:
Could you elaborate on this a little bit? I don't see a get_loc test in that file.